doc/release-notes-162-163.md

```diff
@@ -0,0 +1,15 @@
+GUI changes
+-----------
+
+- The Peers window has a new sortable Network column and the peer details area
+  has a new Network row, both of which display the type of network the peer is
+  connected through: IPv4, IPv6, or Onion, along with the ability to display two
+  potential future additions, I2P and CJDNS. (#162)
+
+- In the Peers window, the NodeId and Node/Service column headers are renamed to
+  Peer Id and Address, respectively. (#162)
+
+- In the Peers window, the Direction field in the peer details area is replaced
+  by a Connection Type field that displays both the direction and the type of
+  peer connection. For more info, place the cursor over the field to see the
+  detailed tooltip. (#163)
```

Comment:
Not sure if we need this. Most people don't read the release notes and making them longer with gui changes, which are (should aim to be) self-explanatory in most cases, doesn't make them more attractive. No strong opinion, though.

Reply:
Yes, I wasn't sure either. I also didn't check if the GUI PR numbers work with the scripts.

Reply:
Dropped the release notes, can add them later on if anyone wants.